components/eam/src/physics/cam/physics_types.F90

```diff
@@ -103,7 +103,23 @@ module physics_types
           te_ini,  &! vertically integrated total (kinetic + static) energy of initial state
           te_cur,  &! vertically integrated total (kinetic + static) energy of current state
           tw_ini,  &! vertically integrated total water of initial state
-          tw_cur    ! vertically integrated total water of new state
+          tw_cur,  &! vertically integrated total water of new state
+          tc_curr,    &! vertically integrated total carbon of current state
+          tc_init,    &! vertically integrated total carbon at start of run
```

Comment:
This is very minor, but the old variable te_ini, I think, is intended to be used as TE per physics time step, not per simulation (though it is probably not used at all). now there is tc_init, with a very similar name, but it is a variable for the initial state of the whole simulation.  Just. a name convention to be aware of.

Reply:
Yea... that is a bit confusing.  Any suggestions on a change?  I selected the "_init" suffix because it seemed to be in line with how "init" gets used in other pieces of code, but I am open to changing it if it helps avoid confusion in the future.